addons/dark-www/experimental_scratchwww.css

```diff
@@ -221,6 +221,7 @@ body:not(.sa-body-editor) .button.white,
 .tabs li.active:hover,
 .grid .thumbnail .thumbnail-info .thumbnail-title .thumbnail-creator a,
 .unsupported-browser .faq-link-text,
+a.social-messages-profile-link,
 a.social-messages-profile-link:link,
```

Comment:
Add `a.social-messages-profile-link:visited` here and remove the `:not()`s under "Link color" - they make selectors more specific, which will probably cause problems.